portfolio/src/main/java/com/google/sps/data/CalendarGroup.java

```diff
@@ -0,0 +1,126 @@
+package com.google.sps.data;
+
+import java.time.Instant;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.Comparator;
+import java.util.List;
+
+/** This class models a group of calendar events. */
+public class CalendarGroup {
```

Comment:
Based on the comment, consider renaming to CalendarEventsGroup.

Reply:
Updated.